hbase-server/src/main/java/org/apache/hadoop/hbase/master/MasterServices.java

```diff
@@ -552,4 +553,9 @@ default SplitWALManager getSplitWALManager(){
    * @return The state of the load balancer, or false if the load balancer isn't defined.
    */
   boolean isBalancerOn();
+
+  /**
+   * @return the {@link QuotaObserverChore}
+   */
+  public QuotaObserverChore getQuotaObserverChore();
```

Comment:
@saintstack , @apurtell , @Apache9 can you shed some light on what criteria is used to decide what gets added to `MasterServices`? I have similar questions for my own PRs. Thanks.